src/styles/main.scss

```diff
@@ -0,0 +1,55 @@
+$calendar-day-size: 100px;
+$calendar-gap-size: 1px;
+$calendar-font-size: 30px;
+$weekdays: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+
+.calendar {
+  width: 7 * ($calendar-day-size + $calendar-gap-size);
```

Comment:
If the gap exists only between days, a calendar row would be 7 * day-size + 6 * gap-size